test/system/test_cvp_client.py

```diff
@@ -31,6 +31,13 @@
 #
 
 ''' System test for the CvpClient class
+
+    Requirements for CVP Node:
+    1) Just need one node for test with the following account:
```

Comment:
cvprac, like pyeapi and rbeapi, are a component of something bigger. I did not go the .conf route instead letting the application solve this problem. For the ServiceNow CVP integration there is a .conf file for the integration that contains ServiceNow and CVP login information. For the Monitor and Management app the creds come from logging into the app.